cartographer_ros/CMakeLists.txt

```diff
@@ -18,7 +18,6 @@ project(cartographer_ros)
 
 set(PACKAGE_DEPENDENCIES
   cartographer_ros_msgs
-  eigen_conversions
```

Comment:
Why did you remove the dependency? Is it not needed?

Reply:
This change is coming from the base master. My branch is 2 commits ahead of this one, because I branched of master.
https://github.com/cartographer-project/cartographer_ros/pull/1278